src/Header.tsx

```diff
@@ -36,7 +55,18 @@ export function LeftNav({ children, className, hideMobile, ...rest }) {
   );
 }
 
-export function RightNav({ children, className, hideMobile, ...rest }) {
+interface RightNavProps {
+  children?: ReactNode;
+  className?: string;
+  hideMobile?: string;
+}
+
+export function RightNav({
+  children,
+  className,
+  hideMobile,
+  ...rest
```

Comment:
there is no`rest` in RightNavProps it seems to be the leftover params for a `div` so maybe extend  RightNavProps from `HTMLAttributes<HTMLDivElement>`